src/test/java/com/artipie/asto/StorageDeleteAllTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/asto/LICENSE.txt
+ */
+package com.artipie.asto;
+
+import com.artipie.asto.blocking.BlockingStorage;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Tests for {@link Storage#deleteAll(Key)}.
+ *
+ * @since 1.9.1
+ * @todo #348:30min Fix error when trying to save data with prefixed Key.
+ *  Test below goes in error when we try to save data in {@link com.artipie.asto.fs.FileStorage}
+ *  and {@link com.artipie.asto.fs.VertxFileStorage} with prefixed key, after saving data with
+ *  key prefix. Please, fix it and enable this test.
```

Comment:
This todo is not valid - it's not possible to store data in storage with prefixes in file-system: you have file `one` and then you're trying to save file `one/two` using `one` as directory, so it's not possible to fix such scenario.

Reply:
@g4s8 Ok